blocks/init/src/Blocks/components/paragraph/docs/story.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { getExample, props, getOptions } from '@eightshift/frontend-libs/scripts/editor';
```

Comment:
The function `getOptions` is sometimes imported from `/editor`(here) and sometimes from `/scripts`(blocks/init/src/Blocks/custom/card/components/card-options.js)

Is that suppose to be like that or?

Reply:
in general from now on you can load everything from scripts. Please check the slack channel from today.